types/taiko/index.d.ts

```diff
@@ -0,0 +1,366 @@
+    // https://taiko.gauge.org/#mouseaction
+    export function mouseAction(action: 'press' | 'move' | 'release', coordinates: MouseCoordinates, options?: NavigationOptions): Promise<void>;
+    // https://taiko.gauge.org/#scrollto
+    export function scrollTo(selector: SearchElement, options?: EventOptions): Promise<void>;
+    // https://taiko.gauge.org/#scrollright
+    export function scrollRight(selector?: SearchElement | number, px?: number): Promise<void>;
+    // https://taiko.gauge.org/#scrollleft
+    export function scrollLeft(selector?: SearchElement | number, px?: number): Promise<void>;
+    // https://taiko.gauge.org/#scrollup
+    export function scrollUp(selector?: SearchElement | number, px?: number): Promise<void>;
+    // https://taiko.gauge.org/#scrolldown
+    export function scrollDown(selector?: SearchElement | number, px?: number): Promise<void>;
+    // https://taiko.gauge.org/#screenshot
+    export function screenshot(options?: ScreenshotOptions, ...args: SearchElement[]): Promise<Buffer>;
+    // https://taiko.gauge.org/#tap
+    export function tap(selector: SearchElement, options?: TapOptions, ...args: SearchElement[]): Promise<void>;
+
+    /**
+     * Selectors
+     */
+
+    // https://taiko.gauge.org/#dollar
+    export function $(selector: string, ...args: SearchElement[]): Selector;
+    // https://taiko.gauge.org/#image
+    export function image(selector: SearchElement, ...args: SearchElement[]): SearchElement;
+    // https://taiko.gauge.org/#link
+    export function link(selector: SearchElement, ...args: SearchElement[]): SearchElement;
+    // https://taiko.gauge.org/#listitem
+    export function listItem(selector: SearchElement, ...args: SearchElement[]): SearchElement;
+    // https://taiko.gauge.org/#button
+    export function button(selector: SearchElement, ...args: SearchElement[]): SearchElement;
+    // https://taiko.gauge.org/#inputfield
+    export function inputField(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#filefield
+    export function fileField(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#textbox
+    export function textBox(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#combobox
+    export function comboBox(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#dropdown
+    export function dropDown(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#checkbox
+    export function checkBox(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#radiobutton
+    export function radioButton(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#text
+    export function text(selector: string, ...args: SearchElement[]): ElementWrapper;
```

Comment:
can take option that says {exactMatch:true|false}

Reply:
added